### DISCUSSION TITLE
Do we consider supabase as an ORM?

### DISCUSSION THREAD

**Discussion by sandbox-apps** [supabase/supabase#2482] [Questions] [answered]
Or is just a plain client?

**Comment by silentworks** [comment, marked as answer]
I wouldn't say it's an ORM, we don't define objects to create relationships and work with our data. it's just a client library I would say.

**Reply by sandbox-apps** [reply]
Yes you're right that it is also suffering in a SQLish problem where the code are wrap into a string.
